server/api/api/endpoints/model_monitoring.py

```diff
@@ -50,9 +53,19 @@ def __post_init__(self) -> None:
 
 
 async def _verify_authorization(
-    project: str, auth_info: mlrun.common.schemas.AuthInfo
+    project: str, auth_info: mlrun.common.schemas.AuthInfo, client_version: str
 ) -> None:
     """Verify project authorization"""
+    if (
+        semver.Version.parse(client_version)
+        < semver.Version.parse(MINIMUM_CLIENT_VERSION_FOR_MM)
+        or "unstable" in client_version
```

Comment:
why to exclude `unstable`?

Reply:
I assume to make it easy to run tests from your ide/laptop.
@davesh0812, don't you need to do the same in `should_redeploy_monitoring_stream()`?

Reply:
For testing, like @assaf758 said:)
About - `should_redeploy_monitoring_stream()` - I removed this function so I don't sure what you mean.